CHANGELOG.md

```diff
@@ -5,6 +5,17 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+- The interface for a generic (type-based) asynchronous cache dependency.
+  While this doesn't introduce any new implementation(s), this interface has been integrated into
+  Tanjun (based on Hikari types) in places which are currently making cache/GET calls and can be used to
+  better integrate 3rd-party caches with Tanjun.
+  A Redis based integration of this may be found in [hikari-sake](https://github.com/FasterSpeeding/Sake)
+  V1.0.1a1 which is exposed by `RedisResource.add_to_tanjun`.
```

Comment:
be more specific about what the redis based integration is plus add a similar note to the actual docs